DNN Platform/Modules/HTML/Providers/DataProviders/SqlDataProvider/10.00.00.SqlDataProvider

```diff
@@ -161,11 +384,9 @@ BEGIN
 END
 GO
 
--- Drop Procedure GetHtmlTextLog if it exists
+-- Drop Procedure GetHtmlTextLog
 IF EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'{databaseOwner}{objectQualifier}GetHtmlTextLog') AND type in (N'P', N'PC'))
-    BEGIN
-        DROP PROCEDURE {databaseOwner}{objectQualifier}GetHtmlTextLog;
-    END
+    DROP PROCEDURE {databaseOwner}{objectQualifier}GetHtmlTextLog;
 GO
 
 -- Create Procedure GetHtmlTextLog
```

Comment:
The table aliases in the `SELECT` list in this procedure and the next need to include `{objectQualifier}`